src/prefect/server/schemas/core.py

```diff
@@ -581,7 +581,7 @@ class Deployment(ORMBaseModel):
         description="The last time the deployment was polled for status updates.",
     )
     parameter_openapi_schema: Optional[Dict[str, Any]] = Field(
-        default=None,
+        default_factory=dict,
```

Comment:
@desertaxle changed this to get [this](https://github.com/PrefectHQ/prefect/blob/main/tests/test_flows.py#L4002) test to pass.

Reply:
I think this makes sense since it will allow us to pass validation of the schema itself in the default case.